src/redux/reducer.ts

```diff
@@ -55,10 +55,18 @@ function formReducer (state: Information | undefined, action: Actions): Informat
       const newDependents = [...(newState.taxPayer?.dependents ?? [])]
       newDependents.splice(action.formData, 1)
 
+      const filingStatus = (() => {
```

Comment:
This function should probably be called something other than filingStatus to avoid confusion with the FilingStatus constant

Reply:
See how it’s just a value though that gets used in the dictionary on 69

Reply:
It’s just a value though that gets used on L69?

Reply:
I guess it could just be personal preference. I'll resolve for now